app/views/users/passwords/edit.html.slim

```diff
@@ -0,0 +1,37 @@
+.header
+  .std-heading
+    i.fa.fa-key style='color: #444444'
+    = ' ' + t('users.change_pwd')
```

Comment:
The new forms contain artifacts from the old ones for a unified look and feel. There is potential for refactorings in the views but I skipped that for now.

Reply:
Yes, these views could get refactored, I acknowledge the need :)

Reply:
When are these views (`app/views/users/passwords/*`) being used? Probably when resetting the password, but why do we need `edit` and `new`?

Reply:
One is the form to request a password reset, the other to actually reset it after clicking the link in the email.

Reply:
Ah, thanks. `new` asks for the mail address (to receive a reset token via mail) and `edit` allows the user to change the password with a reset token